### PR TITLE
Add file-loader and raw-loader rules

### PR DESCRIPTION
**Pull request by cranberyxl**
Now as a major version bump

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 CI_BUILD_NUMBER ?= $(USER)-snapshot
-VERSION ?= 1.5.$(CI_BUILD_NUMBER)
```
Comment (mattkime) [inline Makefile]: This should be a major version bump, right?

**Comment by mattkime** [comment]
I'd prefer to hold off on merging this until we're certain that we can sync merging this PR with the changes necessary in the consuming repos. Otherwise other more immediate needs might get hung up - such as translation related changes.

**Comment by cranberyxl** [comment]
@mattkime You have to update the lib to get the new changes. I'll do it in a PR in `mup-web` and `pro-web` is on a really old version of cli anyway.

File: Makefile (hunk 1 above)
Comment (cranberyxl) [reply Makefile]: You think this deserves a 2.0? cc: @mmcgahan

Comment (mmcgahan) [reply Makefile]: yeah, since it's a breaking change for consumers that use inline loaders, good make that clear with semver

**Pull request merged**